modules/swagger-codegen/src/main/resources/LoopBackSwift/model.mustache

```diff
@@ -0,0 +1,46 @@
+{{#models}}{{#model}}import ObjectMapper
+{{#useRealm}}
+import Realm
+import RealmSwift
+{{/useRealm}}
+
+public class {{classname}} : {{#useRealm}}Object, {{/useRealm}}Mappable, CustomStringConvertible {
```

Comment:
Is it possible to add comments to the templates that don't get emitted into the generated codes? If so, it would be most helpful to have comments added to the templates - including some examples of what the code looks like when generated.